packages/cursorless-vscode/src/keyboard/grammar/generated/grammar.ts

```diff
@@ -0,0 +1,92 @@
+// Generated automatically by nearley, version 2.20.1
+// http://github.com/Hardmath123/nearley
+// Bypasses TS6133. Allow declared but unused functions.
+// @ts-ignore
+function id(d: any[]): any { return d[0]; }
+declare var makeRange: any;
+declare var relative: any;
+declare var simpleAction: any;
+declare var vscodeCommand: any;
+declare var simpleScopeTypeType: any;
+declare var color: any;
+declare var shape: any;
+declare var combineColorAndShape: any;
+declare var direction: any;
+declare var digit: any;
+
+import { capture, command } from "../grammarHelpers"
+import { lexer } from "../lexer";
+
+interface NearleyToken {
+  value: any;
+  [key: string]: any;
+};
+
+interface NearleyLexer {
+  reset: (chunk: any, info: any) => void;
+  next: () => NearleyToken | undefined;
+  save: () => any;
+  formatError: (token: any, message: string) => string;
+  has: (tokenType: any) => boolean;
+};
+
+interface NearleyRule {
```

Comment:
So nearly PokeyRule.

patch it to make it so! a lovely easter egg...

(not sure if I'm joking)